### ISSUE TITLE
Uncaught ReferenceError: text is not defined

### ISSUE THREAD

**Issue by KevinMcCoy** [jamesmacfie/space-tab#16]
I have tried changing
spaceCount = editor.getTabLength()
to
spaceCount = atom-text-editor.getTabLength()

( and all other editor.... to atom-text-editor...  on this one file )

[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.207.0
**System**: Mac OS X 10.10.3
**Thrown From**: [space-tab](https://github.com/jamesmacfie/space-tab) package, v0.4.1

### Stack Trace

Uncaught ReferenceError: text is not defined

```
At /Users/kevin.mccoy/.atom/packages/space-tab/lib/space-tab.coffee:12

ReferenceError: text is not defined
  at Object.module.exports.convertToTabs (/Users/kevin.mccoy/.atom/packages/space-tab/lib/space-tab.coffee:12:20)
  at atom-workspace.<anonymous> (/Users/kevin.mccoy/.atom/packages/space-tab/lib/space-tab.coffee:3:72)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -1:05.1.0 space-tab:convert-to-tabs (atom-text-editor.editor.is-focused)
     -0:08.4.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:07.6.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:04.0 core:save (atom-text-editor.editor.is-focused)
     -0:00.4.0 space-tab:convert-to-tabs (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ],
    "disabledPackages": [
      "autoclose-html",
      "true-color",
      "svn"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "Thumbs.db",
      "config.codekit",
      ".codekit-cache",
      "bower_components",
      "node_modules"
    ],
    "followSymlinks": false
  }
}
```

### Installed Packages

```coffee
# User
autohide-tree-view, v0.20.1
language-cfml, v0.6.0
space-tab, v0.4.1
webbox-color, v0.5.7

# Dev
No dev packages
```

**Comment by jamesmacfie** [comment]
Fixed in 0.4.2

**Comment by KevinMcCoy** [comment]
Thanks!
